curp/src/server/mod.rs

```diff
@@ -48,6 +54,7 @@ mod curp_node;
 mod storage;
 
 /// Default server serving port
+#[cfg(not(madsim))]
```

Comment:
Why should we add `#[cfg(not(madsim))]` here?

Reply:
because it only used by the `run` function, and the function is not enabled in madsim.